sortinghat/core/api.py

```diff
@@ -565,6 +568,73 @@ def add_domain(ctx, organization, domain_name, is_top_domain=True):
     return domain
 
 
+@django.db.transaction.atomic
+def add_group(ctx, organization, parent_name, group_name):
```

Comment:
Because `organization` and `parent_name` can be None, reorganize the definition as explained before.

Reply:
Done 👍